compiler/codegen/cg-symbol.cpp

```diff
@@ -86,7 +86,10 @@
 ********************************* | ********************************/
 
 // these are sets of astrs
-static std::set<const char*> llvmPrintIrNames;
+// Chapel function names requested to be disassembled, and whether they've been
+// matched to C symbol names.
+static std::unordered_map<const char*, bool> llvmPrintIrNames;
```

Comment:
Is it possible to clear both of these after disassembly has completed?

Reply:
It would be, but I think the value would be minimal since all the contained `const char*`s are `astr`s with global lifetime anyways.